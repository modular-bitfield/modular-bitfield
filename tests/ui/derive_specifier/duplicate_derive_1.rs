use modular_bitfield::prelude::*;

#[bitfield]
#[derive_const(Specifier)]
#[derive( Specifier)]
pub struct SignedInt {
    sign: bool,
    value: B31,
}

fn main() {}
