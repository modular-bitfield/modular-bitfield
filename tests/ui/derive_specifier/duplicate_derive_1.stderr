error: encountered duplicate `#[derive_const(Specifier)]` parameter
 --> tests/ui/derive_specifier/duplicate_derive_1.rs:4:21
  |
4 | #[derive_const(Specifier)]
#[derive( Specifier)]
  |                     ^^^^^^^^^

error: previous `#[derive_const(Specifier)]` parameter here
 --> tests/ui/derive_specifier/duplicate_derive_1.rs:4:10
  |
4 | #[derive_const(Specifier)]
#[derive( Specifier)]
  |          ^^^^^^^^^
